.github/workflows/changelog.yml

```diff
@@ -9,23 +9,18 @@ jobs:
     runs-on: ubuntu-latest
     steps:
 
-      - name: Check PR Description
-        id: check_description
+      - name: Checkout code
+        uses: actions/checkout@v2
+      
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
+
+      - name: Install dependencies
         run: |
-          description=$(jq -r ".pull_request.body" "$GITHUB_EVENT_PATH" | awk '/## Changelog/{flag=1; next} /##/{flag=0} flag' | grep -oP '(?<=- \*\*Description:\*\* ).*' | sed 's/^[[:space:]]*//;s/[[:space:]]*$//')
-          echo "$description"
-          default_description="Summary of change(s)"
-          if [ -n "$description" ] && [ "$description" != "$default_description" ]; then
-            echo "containsChangelog=true" >> $GITHUB_OUTPUT
-          else
-              echo "containsChangelog=false" >> $GITHUB_OUTPUT
-          fi
-      - name: Results
+          pip install github
```

Comment:
I don't see this actually being used anywhere? This dependency installing seems to be breaking on github actions, so I think you can probably remove this?